internal/commands/vaultsecrets/secrets/create_test.go

```diff
@@ -45,12 +50,21 @@ func TestNewCmdCreate(t *testing.T) {
 			Name:    "Failed: No secret name arg specified",
 			Profile: testProfile,
 			Args:    []string{},
-			Error:   "ERROR: accepts 1 arg(s), received 0",
+			Error:   "ERROR: missing required flag: --data-file=DATA_FILE_PATH",
 		},
 		{
 			Name:    "Good: Secret name arg specified",
 			Profile: testProfile,
-			Args:    []string{"test"},
+			Args:    []string{"test", "--data-file=DATA_FILE_PATH"},
```

Comment:
Could you please show an example of what a data file for Twilio/MongoDB would look like (probably worth adding it to the PR description and maybe the documentation in another PR).

Reply:
+1